client/src/components/utils_sasCode.ts

```diff
@@ -0,0 +1,57 @@
+// Copyright © 2023, SAS Institute Inc., Cary, NC, USA.  All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { window, workspace, ColorThemeKind } from "vscode";
+
+export function wrapCode(code: string): string {
+  const outputHtml = !!workspace
+    .getConfiguration("SAS")
+    .get("session.outputHtml");
+  const htmlStyle: string = workspace
+    .getConfiguration("SAS")
+    .get("results.html.style");
+
+  if (outputHtml) {
+    let odsStyle = "";
+    switch (htmlStyle) {
+      case "(auto)":
+        switch (window.activeColorTheme.kind) {
+          case ColorThemeKind.Light: {
+            odsStyle = "Illuminate";
+            break;
+          }
+          case ColorThemeKind.Dark: {
+            odsStyle = "Ignite";
+            break;
+          }
+          case ColorThemeKind.HighContrast: {
+            odsStyle = "HighContrast";
+            break;
+          }
+          case ColorThemeKind.HighContrastLight: {
+            odsStyle = "Illuminate";
+            break;
+          }
+          default: {
+            odsStyle = "";
```

Comment:
Do we need `odsStyle = "";` here since we started with an empty string up top?

Reply:
I didn't have the default section there initially for that reason. I added it at @scnwwu's suggestion. Perhaps a little more readable.